docs/code-quality/clang-tidy.md

```diff
@@ -31,14 +31,51 @@ For more information, see [How to: Set Code Analysis Properties for C/C++ Projec
 
 ## CMake
 
-In CMake projects, you can configure Clang-Tidy checks within *`CMakeSettings.json`*. Once opened, select "Edit JSON" in the top right-hand corner of the CMake Project Settings Editor. The following keys are recognized:
+In CMake projects, you can configure Clang-Tidy checks within *`CMakeSettings.json`* or *`CMakePresets.json`*. 
+
+The following keys are recognized for clang-tidy:
 
 - `enableMicrosoftCodeAnalysis`: Enables Microsoft Code Analysis
 - `enableClangTidyCodeAnalysis`: Enables Clang-Tidy analysis
 - `clangTidyChecks`: Clang-Tidy configuration, specified as a comma-separated list, that is, checks to be enabled or disabled
 
 If neither of the "enable" options are specified, Visual Studio will select the analysis tool matching the Platform Toolset used.
 
+### CMake Settings
+To edit your clang-tidy settings, open your CMake Settings, select "Edit JSON" in the top right-hand corner of the CMake Project Settings Editor. You can use the keys above to fill out your clang-tidy specifications in the CMake Settings json file. 
```

Comment:
We make UI bold, e.g. not "Edit JSON" but **Edit JSON**.
We don't say where it is (top right-hand corner) because we find that UI descriptions like that aren't durable.